Userland/Utilities/mknod.cpp

```diff
@@ -5,36 +5,36 @@
  */
 
 #include <AK/Format.h>
+#include <LibCore/ArgsParser.h>
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
 #include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/sysmacros.h>
 
-static int usage()
-{
-    warnln("usage: mknod <name> <c|b|p> [<major> <minor>]");
-    return 1;
-}
-
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     TRY(Core::System::pledge("stdio dpath"));
 
+    StringView name;
+    StringView type_string;
+    StringView major_string;
+    StringView minor_string;
+
+    Core::ArgsParser args_parser;
+    args_parser.set_general_help("Creates a file system node.");
+    args_parser.add_positional_argument(name, "Pathname to create", "name", Core::ArgsParser::Required::Yes);
+    args_parser.add_positional_argument(type_string, "Type of file to create", "type", Core::ArgsParser::Required::Yes);
```

Comment:
The help output lost its ability to communicate which types are supported (`c|b|p`). Maybe add this back to the description?